django_fido/forms.py

```diff
@@ -19,6 +19,7 @@ class Fido2RegistrationForm(forms.Form):
                                   widget=forms.HiddenInput)
     attestation = forms.CharField(error_messages={'required': _("Operation wasn't completed.")},
                                   widget=forms.HiddenInput)
+    user_handle = forms.CharField(widget=forms.HiddenInput)
```

Comment:
This field should not be required or possibly dependent on `resident_key` settings.

Reply:
Initially I had this field not required and it seemed dependent on the `resident_key`, but I realised this not the case, user id is always present during registration process, the `resident_key` merely telling the yubikey to store the credential on the key. 

In short, both features are required to make passwordless to work but they are independent on their own.

Reply:
OK, I had no idea about that.